File: PHP/uberX.php
<?php
    class uberX extends Car{
        public $brand;
        public $model;

//Colocando el constructor de la clase
    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
      }

      
    }




?>

File: Java/Uberx.java
class Uberx extends Car{
    String brand;
    String model;

    //Colocando el constructor con la clase padre Car
    public Uberx(String license,Account driver,String brand,String model){
        super(license, driver);
        this.brand=brand;
        this.model=model;


    }
}
